build/Create-ModuleMapping.ps1

```diff
@@ -14,12 +20,12 @@ function Get-DirectoryFileMap {
     # Determine the root directory and the output based on the Source parameter
     switch ($Source) {
         'Entra' {
-            $RootDirectory = "../module_legacy/Entra/Microsoft.Graph.Entra/"
-            $OutputDirectory = '../module_legacy/Entra/config/'
+            $RootDirectory = (Join-Path $PSScriptRoot "../module/Entra/Microsoft.Entra/")
```

Comment:
source folder is `module_legacy` not `module`

Reply:
This function was added for the sake of creating a reverse mapping for auditing..i.e. given a moduleMapping.json,  and the already split files, we can create a reverse mapping (from module structure to a moduleMapping.json file) and check if the two match i.e. the moduleMapping.json is the single source of truth used for splitting..